packages/open-payments/src/client/index.ts

```diff
@@ -42,17 +42,39 @@ export interface RouteDeps extends BaseDeps {
 }
 
 export interface UnauthenticatedResourceRequestArgs {
+  /**
+   * The full URL of the requested resource.
+   *
+   * For example, if the requested resource is an incoming payment:
+   * ```
+   * https://openpayments.guide/alice/incoming-payments/08394f02-7b7b-45e2-b645-51d04e7c330c`
+   * ```
+   */
   url: string
 }
 
 interface AuthenticatedRequestArgs {
+  /**
+   * The access token required to access the resource.
+   * This token is provided when a grant is created.
+   *
+   * @see [Open Payments - Grant Request](https://docs.openpayments.guide/reference/post-request)
```

Comment:
Are we gonna generate some JSDocs?

Reply:
yes, I was planning on using [typedoc](https://typedoc.org/guides/overview/), though I'm not 100% sure on whether  to deploy the pages somewhere, or just keep them available locally only